index.html

```diff
@@ -29,5 +29,122 @@ <h4>CONNECT WITH US</h4>
+                <li class="b-e-d">Back End Dev</li>
+                <li class="year">2022</li>
+            </ul>
+            <p  class="work1-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="framework">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+            </div>        
+        </section>
+
+        <section class="work-1">
+            <div class="work-img"><img src="./assets/images/Snapshoot-Portfolio1.png" alt="Portfolio snapshoot"></div>
+            <div class="work-desc"><h1>Multi-Post
+                Stories</h1>
+            <ul class="frame">
+                <li class="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2022</li>
+            </ul>
+            <p  class="work1-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="framework">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+            </div>        
+        </section>
+
+        <section class="work-1">
+            <div class="work-img"><img src="./assets/images/Snapshoot-Portfolio2.png" alt="Portfolio snapshoot"></div>
+            <div class="work-desc"><h1>Tonic</h1>
+            <ul class="frame">
+                <li class="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2022</li>
+            </ul>
+            <p  class="work1-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="framework">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+            </div>        
+        </section>
+
+        <section class="work-1">
+            <div class="work-img"><img src="./assets/images/Snapshoot-Portfolio3.png" alt="Portfolio snapshoot"></div>
+            <div class="work-desc"><h1>Multi-Post
+                Stories</h1>
+            <ul class="frame">
+                <li class="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2022</li>
+            </ul>
+            <p  class="work1-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="framework">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+            </div>        
+        </section>
+        
+        <section class="description-1">
+            <h1>About<br>
+                Myself</h1>
```

Comment:
- [ ] `h1` is the top header of the page, and should be used once. `h2-6` are subheaders and can be used repeatedly. At least that's how I see it. Kindly, try to refactor that by using h2 tag for example